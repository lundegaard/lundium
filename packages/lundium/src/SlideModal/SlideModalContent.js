import React from 'react';
import PropTypes from 'prop-types';
import { cx } from 'ramda-extension';

import Box from '../Box';

const SlideModalContent = ({ className, ...otherProps }) => (
	<Box className={cx('slide-modal__content', className)} {...otherProps} />
);

SlideModalContent.propTypes = {
	/** Class name for root. */
	className: PropTypes.string,
};

export default SlideModalContent;
