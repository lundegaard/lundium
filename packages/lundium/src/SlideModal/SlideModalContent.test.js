import React from 'react';
import { shallow } from 'enzyme';

import SlideModalContent from './SlideModalContent';

describe('<SlideModalContent />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<SlideModalContent {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
