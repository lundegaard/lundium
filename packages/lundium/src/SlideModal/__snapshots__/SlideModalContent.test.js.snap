// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SlideModalContent /> matches snapshot when shallowly rendered 1`] = `
<forwardRef(Box)
  className="slide-modal__content"
/>
`;
